src/Umbraco.Core/Constants-SystemDirectories.cs

```diff
@@ -42,7 +44,11 @@ public static class SystemDirectories
             public const string Install = "~/install";
 
             public const string AppPlugins = "/App_Plugins";
-            public static string AppPluginIcons => "/Backoffice/Icons";
+
+            [Obsolete("Use PluginIcons instead")]
+            public const string AppPluginIcons = "/Backoffice/Icons";
```

Comment:
Changing a static property to a const is a breaking change. I'll have to reintroduce it as static property

Reply:
Woops sorry @bergmania, I did not pay attention to that, I'll be more attentive next time 😬
Do I need to do anything to revert?

Reply:
All good, I will handle on the release branch. There is a couple of breaking changes that needs to be fixed :)